backend/utils.py

```diff
@@ -33,3 +37,42 @@ def create_transaction(db):
             pass
 
     return db.transaction()
+
+
+class AppLogger(object):
+
+    @staticmethod
+    def log_info(args, response):
+        logger.info(
+            f"{reqparse.request.method} - "
+            f"{reqparse.request.endpoint} - "
+            f"{current_user.username} - "
+            f"{current_user.email} - "
+            f"{args} - "
+            f"{response} - "
+            f"{datetime.utcnow().strftime('%d %b, %Y, %H:%M:%S')}"
```

Comment:
timestamp is generally the first entry in the log. suggest moving this to the top.

Reply:
Sure